corehq/apps/locations/templates/locations/manage/location.html

```diff
@@ -79,6 +79,14 @@
     {% if products_per_location_form %}
       <li><a href="#products" data-toggle="tab">{% trans "Products" %}</a></li>
     {% endif %}
+
+    {% if location.get_id %}
+    <li>
+      <a href="{% url "create_location" domain %}?parent={{ location.get_id }}">
+        {% trans "New Child Location" %}
```

Comment:
In [another PR](https://github.com/dimagi/commcare-hq/pull/6232/files#diff-4176606c2167bdf53663d62ce886ddfcR74) I saw "Sub-Locations". Is a "Child Location" the same as a "Sub-Location"? Should the language be consistent?